components/browser/errorpages/src/main/res/values/strings.xml

```diff
@@ -14,14 +14,11 @@
     ]]>
     </string>
 
-    <string name="error_connect_title">@string/error_connectionfailure_title</string>
-    <string name="error_connect_message">@string/error_connectionfailure_message</string>
+    <!-- The document title for an error page (i.e., the contents of the `<title>` tag). -->
+    <string name="error_host_lookup_title">Server not found</string>
 
-    <string name="error_timeout_title">The connection timed out</string>
-    <string name="error_timeout_message">@string/error_connectionfailure_message</string>
-
-    <string name="error_hostLookup_title">Server not found</string>
-    <string name="error_hostLookup_message"><![CDATA[
+    <!-- In the example, the two URLs in markup do not need to be translated. -->
+    <string name="error_host_lookup_message"><![CDATA[
```

Comment:
I couldn't find where this string was used

Reply:
Yes, there are some strings which aren't being mapped to an`ErrorType` value yet, but we allowed them to be added to resource file to have them translated until we figure out if we should keep or remove it (better to have it translated than not?). We can update our API and release much quicker if we didn't have to wait for new translations if we decide we need it later or simply remove it otherwise.